src/Smp.Web/ClientApp/src/app/messages/messages.component.scss

```diff
@@ -2,11 +2,31 @@
 @import '~sass/header-partial';
 
 .messages-body {
-  height: 100vh;
+  padding: 4rem 0 0 0;
   margin: 0;
-  padding: 0;
 
   .messages-content {
-    padding: 4rem 0 4.3rem 0;
+    height: calc(100vh - 8rem);
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: repeat(12, 1fr);
+
+    .conversation-list {
+      grid-column: span 3;
+
+      .conversation-preview {
+        .profile-picture {
+          display: inline-block;
+          width: 50px;
+          height: 50px;
+        }
+      }
+    }
+
+    .conversation-window {
+      grid-column: span 9;
+      grid-row: span 12;
+      border-left: 2px solid $blue;
```

Comment:
Perhaps there's a better way of doing this, but a border has been (maybe temporarily) added to separate the list of conversations from the conversation window.